x-pack/plugins/ml/server/models/data_recognizer/modules/security_windows/ml/v3_windows_anomalous_service.json

```diff
@@ -32,19 +32,20 @@
     "time_field": "@timestamp"
   },
   "custom_settings": {
-          "job_tags": {
+    "job_tags": {
       "euid": "8007",
       "maturity": "release",
       "author": "@randomuserid/Elastic",
       "version": "3",
       "updated_date": "5/16/2022"
-  },
+    },
     "created_by": "ml-module-security-windows-v3",
```

Comment:
As with the linux jobs, some of the jobs in this module have `-v3` at the end of the `created_by` and some don't. Is this deliberate? I'd vote for them being consistent.

Reply:
Agreed.